types/weixin-app/index.d.ts

```diff
@@ -0,0 +1,2127 @@
+// Type definitions for wx-app 1.0
+// Project: https://mp.weixin.qq.com/debug/wxadoc/dev/api/
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/* =================== USAGE ===================
+1. Create file xxx.ts
+2. import {} from 'weixin-app';
```

Comment:
It is like typing of node, if I want use if in my project, I must import it `import {} from 'weixin-app';` like I import node `import {} from 'node';`